src/classes/groundplaces.ts

```diff
@@ -47,13 +54,21 @@ export class GroundPlacesController {
   }
 
   /**
-   * @description Returns a list of places.
+   * @description Returns a list of ground places.
    * @param {string} query - Can be a name, a Gpuid, a unique name or other name.
-   * @param {AutoCompleteFilters} filters - Filters with different options (StopGroup, StopCluster, Serviced, SegmentProvider).
+   * @param {AutoCompleteFilters[]|undefined} filters - Filters with different options (StopGroup, StopCluster, Serviced, SegmentProvider).
+   *
+   * If you do not give filters, the list will not be filtered.
    * @returns {AutoComplete[]}
    */
   // @ts-ignore
-  public autocomplete(query: string, filters: AutoCompleteFilters[]): AutoComplete[] {}
+  public autocomplete(query: string, filters?: AutoCompleteFilters[]): AutoComplete[] {
```

Comment:
investigate if you can optimize by doing the search and the filter in the same `.fillter()` method to save loop turns. Just an idea, not sure of this

Reply:
I 've set up the filter with search and filtering at the same time, it makes the code more verbose, but this is better this way yes